cpanfile

```diff
@@ -1,7 +1,6 @@
-requires 'Locale::Country::Multilingual';
-
 on configure => sub {
-    requires 'Locale::Country', '3.59';
+    requires 'Locale::Country', '3.42';
```

Comment:
Why degrade the version of `Locale::Country` ?

Reply:
because 3.42 is enough, 3.59 too much change

Reply:
As this was introduced only due to the Perl 5.28 testing, this set of `configure` dependencies can be removed for now.

Reply:
I think it still necessary to keep the dependencies in cpanfile? becaues it do depend on it, even we not testing 5.28 now, but in fetaure or other users maybe need it

Reply:
those require not added by test perl 5.28, it is already before that, but I added the version 3.59 and move it into on configure {} try to turn off the warning